README.md

```diff
@@ -197,6 +198,12 @@ The choice is defined with the variable `kube_network_plugin`. There is also an
 option to leverage built-in cloud provider networking instead.
 See also [Network checker](docs/netcheck.md).
 
+## Ingress Plugins
+
+- [ambassador](docs/ambassador.md): the Ambassador Ingress Controller and API gateway.
```

Comment:
I am wondering we can specify `roles/kubernetes-apps/ingress_controller/ambassador/README.md` instead of `docs/ambassador.md`

Reply:
I think it would be better to either link to an improved documentation about Ambassador in the docs or the external docs in `getambassador.io`,  but I don't think there are any links to READMEs in this file...